cmd/nvidia-mig-manager/main.go

```diff
@@ -211,11 +211,10 @@ func main() {
 		},
 		&cli.StringFlag{
 			Name:        "driver-root",
```

Comment:
If we consider the new value the "correct" value, should we change this to `nvidia-driver-root` and make the alias `driver-root`?

Reply:
Yes. We should.